config/initializers/sidekiq.rb

```diff
@@ -32,7 +32,10 @@
 
 Sidekiq.configure_server do |config|
```

Comment:
There were no existing tests for this initializer file (nor for most initialisers). I'm not convinced writing a test here to check that Sidekiq can start up a job successfully is that worth the time. There's obviously the question of whether the DataExportXMLJob class works correctly (and there's already a test for that) so I guess the final piece would be to check that `config/sidekiq_cron_schedule.yml` is defined correctly, however I don't think that's worth the time investment now (if we do want to do that, I think we should raise a separate ticket for that). Do you agree?

Reply:
I raised a ticket for this then: https://dluhcdigital.atlassian.net/browse/CLDC-2994.

In it I do mention that we might not want tests at all - it's more a placeholder for making the decision, so it doesn't get forgotten.